napari/_qt/qt_main_window.py

```diff
@@ -138,14 +138,17 @@ def current_viewer(cls):
         window = cls.current()
         return window._qt_viewer.viewer if window else None
 
-    def event(self, e):
+    def event(self, e) -> bool:
```

Comment:
```suggestion
    def event(self, e: QEvent) -> bool:
```

Kinda obvious, but since you're updating the return type and we imported `QEvent`, might as well?